AmazonChimeSDK/AmazonChimeSDK/internal/video/DefaultVideoClientController.swift

```diff
@@ -14,22 +14,22 @@ import UIKit
 class DefaultVideoClientController: NSObject {
     var clientMetricsCollector: ClientMetricsCollector
     var logger: Logger
-    var videoClient: VideoClient?
+    var videoClient: VideoClientProtocol?
     var videoSourceAdapter = VideoSourceAdapter()
     var videoClientState: VideoClientState = .uninitialized
     let videoTileControllerObservers = ConcurrentMutableSet()
     let videoObservers = ConcurrentMutableSet()
     var dataMessageObservers = [String: ConcurrentMutableSet]()
 
     private let configuration: MeetingSessionConfiguration
-    private let defaultVideoClient: VideoClient
+    private let defaultVideoClient: VideoClientProtocol
 
     // This internal camera capture source is used for `startLocalVideo()` API without parameter.
     private let internalCaptureSource: DefaultCameraCaptureSource
     private var isInternalCaptureSourceRunning = true
     private let eventAnalyticsController: EventAnalyticsController
 
-    init(videoClient: VideoClient,
+    init(videoClient: VideoClientProtocol,
```

Comment:
Why this change?

Reply:
`VideoClient` is a protocol defined in the ObjC layer and cannot be mocked in Swift. The workaround is to define an identical protocol in Swift and let the type conform to that, which enables mockingbird to generate mock classes automatically based on the protocol thus being able to stub/verify any functions being called or not called with, for example:
```
verify(videoClientMock.sendDataMessage(self.topic, data: any(), lifetimeMs: 0)).wasNeverCalled()
```
See the new [DefaultVideoClientControllerTests](https://github.com/aws/amazon-chime-sdk-ios/blob/8691784f43a4c9ab1c21ebdc5796bc7f64c2992d/AmazonChimeSDK/AmazonChimeSDKTests/internal/video/DefaultVideoClientControllerTests.swift) test added. We did that long time back for AudioClient.